charts/rancher-project-monitoring/0.1.0+up0.0.1/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Project Monitoring
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>=1.16.0-0'
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <=2.6.99-0'
```

Comment:
This will be fixed when we replace it with https://github.com/rancher/prometheus-federator/pull/6